lib/improver/tests/utilities/test_save.py

```diff
@@ -79,7 +79,15 @@ def test_basic_cube(self):
         self.assertTrue(os.path.exists(self.filepath))
 
     def test_basic_cube_list(self):
-        """ Test functionality for saving iris.cube.CubeList """
+        """
```

Comment:
I'm not familiar with these tests, but if we're not able to read the resulting file using IMPROVER load utilities, we probably shouldn't be able to save them either. @benfitzpatrick

Reply:
Thats the topic of ticket IMRPO-507:
https://exxconfigmgmt:6391/browse/IMPRO-507

We rather cheekily did 508 before 507.

Reply:
Ah, thanks. I didn't see notice that one!